tests/stack_test.go

```diff
@@ -426,6 +426,59 @@ func TestStackBackups(t *testing.T) {
 	})
 }
 
+//nolint:paralleltest // mutates environment variables
+func TestDestroySetsEncryptedkey(t *testing.T) {
+	t.Run("SetEncryptedKey", func(t *testing.T) {
```

Comment:
If there's only one subtest, you can encode its name in the test name.
In this case, it's already encoded in the test name so you can just drop the subtest, IMO.